test/app-luatest/checks_test.lua

```diff
@@ -0,0 +1,3 @@
+rawset(_G, '_checks_test_expect_builtin', true)
+
+require('third_party.checks.test.test')
```

Comment:
checks helper is never required, so nothing happens on the second step.

But I'll try to think once more about how we may deal with it more carefully.

Reply:
Luatest requires `test.helper` if there is a file. But it won't implicitly require any other file that could be used as helper, like `third_party.test.helper` in our case.

Reply:
Reworked the approach with asserts (no globals anymore), please see new version here and in checks: https://github.com/tarantool/checks/pull/42/files

Reply:
Regarding one more thing: it may seem that you don't need to assert if checks are built-in while running tests here, since if there are no built-in `checks`, require will fail. Well, it's not. `luatest` brings the `checks` package, so if you do not include this assert and forget to embed the module, everything still will be okay.